packages/react-components/styles/fonts.v2.tsx

```diff
@@ -79,7 +79,7 @@ const fontStyles = StyleSheet.create({
     color: colors.dark,
   },
   large: standards.large,
-  regular: standards.regular,
+  regular: { ...standards.regular },
```

Comment:
Why is this needed?

Reply:
ah leftover from when i tried adding font weight normal here.